src/main/java/net/minestom/server/adventure/audience/PacketGroupingAudience.java

```diff
@@ -58,7 +60,11 @@ default void sendGroupedPacket(@NotNull ServerPacket packet) {
 
     @Override
     default void sendMessage(@NotNull Identity source, @NotNull Component message, @NotNull MessageType type) {
-        Messenger.sendMessage(this.getPlayers(), message, ChatPosition.fromMessageType(type), source.uuid());
+        if (type == MessageType.SYSTEM)
+            Messenger.sendSystemMessage(getPlayers(), message, ChatType.SYSTEM);
+        else
+            Messenger.sendUnsignedMessage(getPlayers(), new MessageSender(Component.text("UNKNOWN SENDER",
+                    NamedTextColor.RED), null), message);
```

Comment:
This should be bracketed imo

Reply:
Yeah it might look better, and iirc there is another one of this somewhere, i'll change that too